mappings/net/minecraft/block/Material.mapping

```diff
@@ -11,8 +11,8 @@ CLASS net/minecraft/class_3614 net/minecraft/block/Material
 	FIELD field_15919 PORTAL Lnet/minecraft/class_3614;
 		COMMENT Material for the various portal blocks.
 	FIELD field_15920 WATER Lnet/minecraft/class_3614;
-	FIELD field_15921 UNUSED_PLANT Lnet/minecraft/class_3614;
-		COMMENT Not in use, but has foliage color.
+	FIELD field_15921 MOSS Lnet/minecraft/class_3614;
+		COMMENT Material for the moss block. Has foliage color.
```

Comment:
MapColor is per blockstate, please don't directly reference the default MapColor a material uses.

Reply:
This is the old docs, so should be changed in another pr i think

Reply:
well this isnt MapColor. this is Material.  moss uses Green as the MapColor
unless im missunderstanding

Reply:
Yes, I'm asking for "Has foliage color" portion to be removed. It was originally added as at the time it was really the only information we had on it, but now we know what it is used for - a Material identical to GOURDS used only for moss.

Reply:
For reference, "foliage color" is what "dark green" MapColor was originally called. Apparently I forgot to update the JD when I renamed it.

Reply:
```suggestion
		COMMENT Material for the moss block.
```
Guess we will do this